data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1924 @@
+
+event "lunarium fleeing rb"
+	system "12 Autumn Above"
+		remove fleet "Heliarch"
+
+event "lunarium detained rb"
+	system "12 Autumn Above"
+		add fleet "Heliarch" 800
+
+
+
+mission "Heliarch Drills 1"
+	name "Refuelling Drill"
+	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in emergency situations."
+	minor
+	source
+		near "Belug" 6 100
+	destination "Belug's Plunge"
+	to offer
+		has "Coalition: Contributor: done"
+		not "Lunarium: Questions: done"
+		random < 8
+		"combat rating" >= 8104
+	on offer
+		conversation
+			`A pair of Heliarch Judicators land by your ship as you leave for the spaceport. The workers storm to the ships, getting ready to refuel them. Coming down from one of the Judicators, a young Saryd with black hair barely reaching her shoulders pulls out a watch of sorts, seeming to time how long they take to completely fill up the ships' fuel tanks.`
+			`	When they're done, she gives an approving nod, and takes a look around the hangars. When she notices you watching, she approaches you. "Greetings, human visitor," she says. "Magister Lotuora, I am. Intrigued by our ships, were you?" You begin a nod, and that's enough for her to keep talking. "A refuelling drill, it was. When coming down, inform the local authorities at the last minute, we do, so that more 'authentic' it is. An important part it is, in testing how well prepared for emergencies we are."`
+			choice
+				`	"What sort of emergencies?"`
+				`	"And you want me to help with some of these drills, I take it?"`
+					goto help
+				`	"It's good that you do that. I should get going now, though."`
+					decline
+			`	"All sorts of emergencies," she responds. "Quarg invasions, criminal attacks, response action to natural disasters, ready for all of them, we need to be."`
+			label help
+			`	She takes a look at your ship, then to the two Judicators. "About done with the refuelling drills, we were. Heading back to <planet>, I was. Some more interesting drills, I could set up, if your assistance, I had. Accompany us back there, would you?"`
+			choice
+				`	"Alright, what do you need me to do?"`
+					goto accept
+				`	"What would these 'more interesting drills' involve?"`
+				`	"Sorry, I'm not interested in that kind of work."`
+					decline
+			`	"Thought of that, I still haven't," she says. "Some ship combat training, potentially. Very beneficial, training with an outsider could be."`
+			choice
+				`	"Alright, so should I just follow your ships now?"`
+				`	"Sorry, but I'm not interested in that kind of work."`
+					decline
+			label accept
+			`	"Follow you to <planet>, my ships will," she says, pointing at the Judicators. "In a rush to head back, we are not, so if immediate business elsewhere, you have, follow you for a while, we could. An extension to the refuelling drills, it would be."`
```

Comment:
```suggestion
			`	"Follow you to <planet>, my ships will," she says, pointing at the Judicators. "In a rush to head back, we are not, so if immediate business elsewhere, you have, follow you for a while, we could. An extension to the refueling drills, it would be."`
```